src/OmnichannelChatSDK.ts

```diff
@@ -660,6 +709,11 @@ class OmnichannelChatSDK {
                 if (members.length === 1) {
                     onAgentEndSessionCallback(message);
                 }
+
+                if (this.isPersistentChat && !this.chatSDKConfig.persistentChat?.disable) {           
+                    clearInterval(this.PersistentChatTimer);
+                    this.PersistentChatTimer = null;
```

Comment:
The logic to `clearInterval` & reset the timer should only happen if `members.length === 1`. ThreadUpdate is triggered on any update in the chat (member added, member removed). So, it makes sense to clear the interval only when there's only one member left in the chat.